packages/compiler/test/helpers/usage-resolver.test.ts

```diff
@@ -124,4 +129,40 @@ describe("compiler: helpers: usage resolver", () => {
 
     deepStrictEqual(usages, { inputs: [], outputs: ["MyEnum"] });
   });
+
+  describe("resolving usage of specific operation", () => {
+    it("only find usage in that operation", async () => {
+      const usages = await getUsages(
+        `
+          model Foo {}
```

Comment:
should include one model outside the op/interface in these tests.  Otherwise, lgtm